src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,90 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final User bob = new User(55555L, "bobr", "123456", 16);
+    private static final User john = new User(4444L, "john1998", "123", 26);
+    private static final User steve = new User(333L, null, "123456", 65);
+    private static final User alice = new User(22L, "alice", "qwerty56", null);
+    private static final User artur = new User(22L, "artur", "123456789", -18);
+    private static final User kris = new User(22L, "kris", null, 18);
+    private static final User kirill = new User(1L, "Kirill", "987654321", 90);
+    private static final User stas = new User(154354L, "Stas", "798564213", 40);
+    private static final User dima = new User(164354L, "Dima", "9413543fg545", 25);
+    private static final User sveta = new User(154352L, "Sveta", "asdfgh456", 31);
```

Comment:
I think this looks a little confusing. You can create a correct User object for each test using @ BeforeEach and then set wrong values for him in every test. (for example: correctUser.setAge(null))